crates/hyperswitch_connectors/src/connectors/aci/transformers.rs

```diff
@@ -94,6 +94,38 @@ impl TryFrom<&ConnectorAuthType> for AciAuthType {
     }
 }
 
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(rename_all = "UPPERCASE")]
+pub enum AciRecurringType {
+    Initial,
+    Repeated,
+    RegistrationBased,
+}
+
+fn get_str(key: &str, metadata: &serde_json::Value) -> Option<String> {
```

Comment:
this utility fn`get_str`  is not needed separately, you can directly use `get` 
https://docs.rs/std_tools/latest/std_tools/struct.Map.html#method.get